ksonnet-gen/kubeversion/data.go

```diff
@@ -72,13 +72,82 @@ var versions = map[string]versionData{
 			"local": "localStorage",
 		},
 		constructorSpecs: map[string][]CustomConstructorSpec{
+			//
+			// Apps namespace.
+			//
+			"io.k8s.kubernetes.pkg.apis.apps.v1beta1.Deployment":         v1beta1Deployment,
+			"io.k8s.kubernetes.pkg.apis.apps.v1beta1.DeploymentList":     objectList,
+			"io.k8s.kubernetes.pkg.apis.apps.v1beta1.DeploymentRollback": v1beta1DeploymentRollback,
+			"io.k8s.kubernetes.pkg.apis.apps.v1beta1.Scale":              v1beta1Scale,
+			"io.k8s.kubernetes.pkg.apis.apps.v1beta1.StatefulSet":        v1beta1StatefulSet,
+			"io.k8s.kubernetes.pkg.apis.apps.v1beta1.StatefulSetList":    objectList,
+
+			//
+			// Extensions namespace.
+			//
+
+			"io.k8s.kubernetes.pkg.apis.extensions.v1beta1.Deployment":         v1beta1Deployment,
+			"io.k8s.kubernetes.pkg.apis.extensions.v1beta1.DeploymentList":     objectList,
+			"io.k8s.kubernetes.pkg.apis.extensions.v1beta1.DeploymentRollback": v1beta1DeploymentRollback,
+			"io.k8s.kubernetes.pkg.apis.extensions.v1beta1.Scale":              v1beta1Scale,
+			"io.k8s.kubernetes.pkg.apis.extensions.v1beta1.StatefulSet":        v1beta1StatefulSet,
+			"io.k8s.kubernetes.pkg.apis.extensions.v1beta1.StatefulSetList":    objectList,
+
+			//
+			// Authentication namespace.
+			//
+
+			"io.k8s.kubernetes.pkg.apis.authentication.v1.TokenReview": []CustomConstructorSpec{
+				newConstructor(
+					"new",
```

Comment:
There seems to be a mandatory "new" as the first param for every call to newConstructor -- what is this for?